tests/unit/test_crud.py

```diff
@@ -226,44 +243,30 @@ def test_get_contact_by_any_id_missing(dbsession, sample_contacts):
     ],
 )
 def test_get_multiple_contacts_by_any_id(
-    dbsession, sample_contacts, alt_id_name, alt_id_value
+    dbsession, email_factory, alt_id_name, alt_id_value
 ):
-    dupe_id = str(uuid4())
-    create_email(
-        dbsession,
-        EmailInSchema(
-            email_id=dupe_id,
-            primary_email="dupe@example.com",
-            basket_token=str(uuid4()),
-            sfdc_id=(
-                alt_id_value if alt_id_name == "sfdc_id" else "other_sdfc_alt_id_value"
-            ),
-        ),
+    """Two contacts can share the same:
+    - fxa primary_email
+    - amo user_id
+    - sfdc_id
+    - mofo_contact_id
```

Comment:
This was interesting to discover.

Reply:
Oh, that means there is no unique constraint on the main email table for this tuple of fields?
Do we know if we have occurences in prod data?

Reply:
Yeah, I'm wondering why they were omitted, but even before, we had tests to assert that this behavior was true, so I'm wondering why it was modeled this way 🤔. Maybe it was intentional?